synapse/lib/cell.py

```diff
@@ -1239,6 +1244,90 @@ async def __anit__(self, dirn, conf=None, readonly=False, parent=None):
         # phase 5 - service networking
         await self.initServiceNetwork()
 
+    async def _storCellHiveMigration(self):
+        versnode = await self.hive.open(('cellvers',))
```

Comment:
We should be able to open these nodes as context managers, since they are Base objects.
`async with await self.hive.open(('cellvers',)) as versnode:`